core/unix/signal_private.h

```diff
@@ -191,7 +191,7 @@ typedef _STRUCT_UCONTEXT /* == __darwin_ucontext */ kernel_ucontext_t;
  * (these are from /usr/src/linux/arch/i386/kernel/signal.c for kernel 2.4.17)
```

Comment:
There are several two-space-indented #includes at the top of this file that were not detected by the rule -- why not?